### ISSUE TITLE
quizlet.com: breakage

### ISSUE THREAD

**Issue by shrimp-guy** [uBlockOrigin/uAssets#22151]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://quizlet.com/871637092/comp-phys-exam-1-flash-cards/`

### Category

breakage

### Description

Removing ads from the page causes elements to become horizontally squashed, making them awkwardly small, if not completely unreadable. Refreshing the page shows the elements start out at their correct width, then shrink after a split second, as if something is adjusting the page to the elements squishier.
Effect applies to all "flashcards" pages on the website. Website sizes elements normally when Ublock is off.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/136673881/897a3a46-c40f-4235-b929-f328089b729d)
![image](https://github.com/uBlockOrigin/uAssets/assets/136673881/137ee61d-9479-4622-ac49-32f17c9c93d6)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.55.0
Firefox: 121
filterset (summary):
 network: 114099
 cosmetic: 44899
 scriptlet: 18772
 html: 2038
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 37209-215, 55m Δ
  ublock-badware: 7789-10, 55m Δ
  ublock-privacy: 992-0, 55m Δ
  ublock-unbreak: 2235-1, 55m Δ
  ublock-quick-fixes: 143-0, 55m Δ
  easylist: 76774-537, 55m Δ
  easyprivacy: 43560-845, 55m Δ
  urlhaus-1: 9080-0, 7m
  plowe-0: 3782-1, 7m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 814 ms (selfie)
 maxAssetCacheWait: 41 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 91
 network:
  adlightning.com: 1
  adnxs.com: 2
  amazon-adsystem.com: 2
  btloader.com: 1
  casalemedia.com: 1
  cloudflare.com: 58
  doubleclick.net: 2
  googletagmanager.com: 1
  indexww.com: 1
  pubmatic.com: 2
  quantserve.com: 1
  rubiconproject.com: 4
  scorecardresearch.com: 1
  sharethrough.com: 6
  sonobi.com: 1
  teads.tv: 1
  yahoo.com: 6
 extended:
  ##.blogads
  ##+js(remove-class, has-sidebar-adz|DashboardPage-inner, div[cla…
  ##+js(remove-class, hasStickyAd, div.hasStickyAd[class^="SetPage…
  ##+js(remove-class, has-adz, div.has-adz, stay)
  ##+js(adjust-setTimeout)
```
</details>

**Comment by stephenhawk8054** [comment]
Caused by

```adb
quizlet.com##+js(remove-class, has-adz, div.has-adz, stay)
```

**Comment by Agshin2004** [comment]
but how to fix that though?

**Comment by stephenhawk8054** [comment]
Just update the filter lists

**Comment by Agshin2004** [comment]
didn't fix the issue, cards are still jammed

[https://quizlet.com/205756017/biology-practice-test-flash-cards](url)

filter list: 
<img width="1001" alt="Screenshot 2024-01-25 at 12 15 59" src="https://github.com/uBlockOrigin/uAssets/assets/85677228/f22b59b4-987f-4ca8-9566-55d2cc93d0d8">

**Comment by stephenhawk8054** [comment]
@Agshin2004 Click on this link and check again: https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-filters&manual=1

Then can you open new tab > reproduce the issue and on that page > click on uBO icon > &#128172; [Report button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > scroll to the bottom > click on `Troubleshooting Information` and copy paste it here?

**Comment by Agshin2004** [comment]
sure,

`uBlock Origin: 1.55.0
Firefox: 122
filterset (summary):
 network: 82431
 cosmetic: 162980
 scriptlet: 27669
 html: 2045
listset (total-discarded, last-updated):
 removed:
  easylist: null
 added:
  adguard-social: 22459-873, 27m
  adguard-cookies: 28451-61, 27m
  adguard-popup-overlays: 26858-1554, 27m
  adguard-mobile-app-banners: 4919-1074, 27m
  adguard-other-annoyances: 14132-277, 27m
  adguard-widgets: 2410-1, 27m
  easylist-annoyances: 4319-122, 27m
  easylist-chat: 180-12, 27m
  fanboy-cookiemonster: 49399-3778, 27m
  easylist-newsletters: 6894-13, 27m
  easylist-notifications: 2908-4, 27m
  fanboy-social: 16111-14, 27m
  [2 lists not shown]: [too many]
 default:
  user-filters: 16-3, never
  ublock-filters: 37272-509, 1m
  ublock-badware: 7795-10, 27m
  ublock-privacy: 988-0, 27m
  ublock-unbreak: 2238-0, 27m
  ublock-quick-fixes: 128-1, 27m
  easyprivacy: 43562-929, 27m
  urlhaus-1: 8745-0, 27m
  plowe-0: 3783-1, 27m
filterset (user): [array of 16 redacted]
trustedset:
 added: [array of 26 redacted]
 removed:
  about-scheme
userSettings:
 autoUpdate: false
 suspendUntilListsAreLoaded: false
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1864 ms (selfie)
 maxAssetCacheWait: 99 ms
 cacheBackend: indexedDB
 unsuspendAfter: 1661 ms
popupPanel:
 blocked: 38
 network:
  adlightning.com: 1
  adnxs.com: 2
  amazon-adsystem.com: 2
  braze.com: 3
  btloader.com: 1
  casalemedia.com: 1
  cloudflare.com: 1
  doubleclick.net: 2
  googletagmanager.com: 1
  id5-sync.com: 1
  indexww.com: 1
  pubmatic.com: 2
  quantserve.com: 2
  rollbar.com: 1
  rubiconproject.com: 4
  scorecardresearch.com: 1
  sharethrough.com: 9
  sonobi.com: 1
  teads.tv: 1
  yahoo.com: 1
 extended:
  ##.d3lcmut
  ##body:style(overflow: auto !important; padding-right: 0 !import…
  ##+js(remove-class, has-adz, div.has-adz, stay)
  ##+js(remove-class, has-sidebar-adz|DashboardPage-inner, div[cla…
  ##+js(remove-class, hasStickyAd, div.hasStickyAd[class^="SetPage…
  ##+js(adjust-setTimeout)`

**Comment by stephenhawk8054** [comment]
@Agshin2004 You are using a lot of custom filter lists. Test in a new browser profile with only uBO and its default settings (update the filter lists there)

**Comment by Agshin2004** [comment]
seems to be working but as soon as I import my filters ui crashes again, is there a way to fix that?

**Comment by stephenhawk8054** [comment]
What do you mean by `crashes`?

**Comment by Agshin2004** [comment]
The cards are being squished (look at the photo that was added by shrimp-guy above)

**Comment by stephenhawk8054** [comment]
> I tried on google chrome with only 1 filter

I don't understand what you were doing here. You mean you disabled all the filter lists and add only 1 filter as a custom filter?

Can you go to `uBlock filters – Ads` list and screenshot the header to here?

**Comment by Agshin2004** [comment]
yes, sure

<img width="888" alt="Screenshot 2024-01-25 at 13 00 09" src="https://github.com/uBlockOrigin/uAssets/assets/85677228/ddd8e0db-0598-4a8c-b350-1e7ce97bdf56">

**Comment by Agshin2004** [comment]
<img width="888" alt="Screenshot 2024-01-25 at 13 03 34" src="https://github.com/uBlockOrigin/uAssets/assets/85677228/5c597c16-c100-475a-8ff2-01d1bf9dabd6">

**Comment by stephenhawk8054** [comment]
@Agshin2004 That does not look like `uBlock filters - Ads` list. It looks like your `My filters` list. Can you screenshot the whole screen?

**Comment by Agshin2004** [comment]
My bad I’m sorry, I’m just looking at it it’s huge (37737 lines), should I delete it all?

**Comment by stephenhawk8054** [comment]
@Agshin2004 No, don't delete anything. I also didn't tell you to disable any default lists.

I just tell you to SCREENSHOT the `uBlock filters - Ads` list

Go to `Filter lists` tab -> Click on the eye icon right next to `uBlock filters – Ads` word -> It will open a new tab with that list -> Screenshot it.

**Comment by shrimp-guy** [comment]
Hey I just wanted to say that @stephenhawk8054 was right about the issue, it was  "quizlet.com##+js(remove-class, has-adz, div.has-adz, stay)"

I fixed the issue with the following steps:
1. Open UBlock Origin on the Quizlet flashcards page, click "Open the logger"
2. Press the "Reload Tab Content" button
3. Go into the "filter logger content" menu and search for scriptlets
4. Click on "quizlet.com##+js(remove-class, has-adz, div.has-adz, stay)"
5. Click on the #@# button in the menu, which should disable it

Attached below is a demonstration of navigating to a different scriptlet (I think there was 4 before, before the problematic one was removed). Refresh the page and you should see the flashcards in their normal resolution.
![image](https://github.com/uBlockOrigin/uAssets/assets/136673881/d5f32d80-0d2e-4aa0-a8f8-71920a9a9232)

**Comment by stephenhawk8054** [comment]
@shrimp-guy You don't need to manually add that. Updating the filter lists didn't work?

**Comment by shrimp-guy** [comment]
I was already poking around in the logger and found those javascript files before posting my complaint, so when you mentioned the specific scriptlet file causing the issue I just turned it off rather than resetting the filter lists. I guess the new filter list could be the reason why it doesn't pop up in the logger anymore.

I do agree that just updating the filter list is the most logical course of action. If that doesn't work however, the steps I perform above DID fix the problem for me.

**Comment by stephenhawk8054** [comment]
@Agshin2004 So looks like when you import your configuration, your filter lists do not have differential update. Can you share your backup file to here?

cc @gorhill

**Comment by Agshin2004** [comment]
here you go: 
[my-ublock-backup_2024-01-25_16.18.15.txt](https://github.com/uBlockOrigin/uAssets/files/14051503/my-ublock-backup_2024-01-25_16.18.15.txt)

**Comment by stephenhawk8054** [comment]
@Agshin2004 Isn't it your new configuration? I mean the one you mentioned here: https://github.com/uBlockOrigin/uAssets/issues/22151#issuecomment-1909667574

**Comment by Agshin2004** [comment]
yes, it is, I don't know why but keep filters clean (my filters) solves the issue

**Comment by stephenhawk8054** [comment]
Yes, that's what I mean

You said

> as soon as I import my filters back

so what were `my filters` that you `import` and caused the issue. It's this one: https://github.com/uBlockOrigin/uAssets/issues/22151#issuecomment-1910091386 ?

**Comment by Agshin2004** [comment]
these ones: 
[my-ublock-static-filters_2024-01-25_12.35.17.txt](https://github.com/uBlockOrigin/uAssets/files/14051730/my-ublock-static-filters_2024-01-25_12.35.17.txt)

**Comment by stephenhawk8054** [comment]
Remove these filters in that file:

```adb
! 2024-01-24 https://quizlet.com
quizlet.com##.d3lcmut
quizlet.com##+js(remove-class, has-adz, div.has-adz, stay)
```

and reset your uBO to default settings

**Comment by RSickenberg** [comment]
Alo, we might want to block also `quizlet.com##.SetPageEmbeddedAd-wrapper`
